src/test/java/foodwhere/testutil/TypicalReviews.java

```diff
@@ -62,4 +64,15 @@ private TypicalReviews() {} // prevents instantiation
     public static List<Review> getTypicalReviews() {
         return new ArrayList<>(Arrays.asList(ALICE, BENSON, CARL, DANIEL, ELLE, FIONA, GEORGE));
     }
+
+    public static AddressBook getTypicalAddressBook() {
```

Comment:
This method is in `TypicalStalls` here, so I don't think this needs to be added.

https://github.com/AY2223S1-CS2103-W14-2/tp/blob/master/src/test/java/foodwhere/testutil/TypicalStalls.java#L61